README.md

```diff
@@ -7,15 +7,14 @@ Amazon ECS presents a low barrier to entry for production-level docker applicati
 
 Broadside offers a simple command-line interface to perform deployments on ECS. It does not attempt to handle operational tasks like infrastructure setup and configuration, which are better suited for tools like [terraform](https://www.terraform.io/).
 
-### Things You Can Do With Broadside
+### Capabilities
 
-- **Deploy a docker image to an ECS cluster and launch a command as an ECS service**
-- **Launch a bash shell directly in the remote docker image** - no messing around with tracking down the server, running `docker ps`, and all the other headaches.
-- **SSH directly onto the server running your image**
-- **Run arbitrary commands in a container that is spun up and spun down as you need it**
-- **Get a lot of status information about your running image**
-- **Tail the logs of a running container**
-- **Horizontally scale an existing deployment to as many instances as your AWS account can handle**
+- Trigger ECS deployments
+- Launch a bash shell directly onto a running container in the cluster
+- SSH directly onto a host running a container
+- Launch an ECS task running an arbitrary command
+- Tail logs of a running container
+- Scale an existing deployment on the fly
```

Comment:
don't think it's necessary, also gives less contrast with the headers when it's bolded